components/ui/icons/ProfessionalIconSystem.tsx

```
@@ -0,0 +1,276 @@
+  Users, BookOpen, Bus, CreditCard, AlertTriangle,
+  GraduationCap, BarChart3, Mail, FileText, Settings,
+  TrendingUp, TrendingDown, CheckCircle, XCircle,
+  RefreshCw, Target, Star, Building2, Lightbulb,
+  Rocket, Phone, MessageCircle, Smartphone, Banknote,
+  User, UserCheck, Folder, Book, Save, ArrowRight,
+  ArrowUpRight, ArrowDownRight, UserPlus, Users2,
+  Calendar, Clock, Activity, Award, BookMarked,
+  Briefcase, Calculator, Camera, ChartBar, ChartLine,
+  ChartPie, Database, Download, Edit, Eye, Filter,
+  Globe, Heart, Home, Info, Layout, List, Lock,
+  LogOut, Map, Monitor, Printer, Search, Shield,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)